### DISCUSSION TITLE
Odyssey class in 2e

### DISCUSSION THREAD

**Discussion by DominusExMachina** [bcholmes/StarTrek2d20#227] [General]
I wanted to create an _Odyssey_-class starship for my gaming group using the creator and the 2e ruleset, but the program won't let me do so unless it's launch date is 2401 (20 years later than when I want to start my campaign, which is 2381 [around the time of _Lower Decks_ seasons 2-4])

When using the 1e ruleset, however, I am still able to do so.

**Comment by bcholmes** [comment]
You're right. It's a mistake on my part. (The Odyssey that appears in Utopia Planetia has a launch date of 2401, but the Odyssey that appears in the GM's Toolkit has a launch date of 2381; looks like I got those dates reversed).

**Reply by DominusExMachina** [reply]
How soon will this get fixed? I'm sure there are a lot of folks - myself included - who'd absolutely **_love_** to get their hands on an _Odyssey_ for games set in the late 24th century.
